db/migrate/20181204180756_change_active_and_permanent_to_not_null.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeActiveAndPermanentToNotNull < ActiveRecord::Migration[5.1]
+  def change
+    change_column_null :passengers, :active, false, true
```

Comment:
Note that I'm assuming that passengers with a `NULL` active state should be made active. That _might_ not be what we want. Any thoughts?

Reply:
Presumably people didn't _deactivate_ them, they just created them, so I think it will depend and we should send a list of the passengers this affects to the ST supervisor like "hey we just activated these passengers: " that way they can do something about it.

Reply:
Whew. There sure are 45 of them:

```
Passenger.where(active: nil).count
   (1.2ms)  SELECT COUNT(*) FROM `passengers` WHERE `passengers`.`active` IS NULL
=> 45
```

Reply:
I'm going to get Tiff and Avia a list